.github/workflows/dbt-checks.yaml

```
@@ -0,0 +1,53 @@
+      - "queries/**/*.sql"
+
+env:
+  DBT_USER: ${{ github.actor }}
+
+jobs:
+  check-files:
+    name: Compile DBT models and format SQL
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)